README.md

```diff
@@ -1,7 +1,7 @@
 <img src="https://travis-ci.org/SkouRene/HL-phpclient.svg?branch=master" /> [![Code Climate](https://codeclimate.com/github/SkouRene/HL-phpclient/badges/gpa.svg)](https://codeclimate.com/github/SkouRene/HL-phpclient) [![codecov](https://codecov.io/gh/SkouRene/HL-phpclient/branch/master/graph/badge.svg)](https://codecov.io/gh/SkouRene/HL-phpclient)
 
 # HL-phpclient
-Php client for heyloyalty api.
+Php client for heyloyalty api. Please note, that this is a backwardcompatible fork of the original client
```

Comment:
Ok no worries, if you want the code merged into the master branch making it compatible with php 5.3 just remove the name change on composer package and the readme note, you will how ever be credited as contributor in the composer file and on the repo. I see no reason why it can not be compatible.